src/components/progress-bar/__test__/index.test.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies
+ */
+import ProgressBar from './../index';
+
+describe( 'The ProgressBar component', () => {
+	it( 'renders without crashing', () => {
+		const div = document.createElement( 'div' );
+		ReactDOM.render( <ProgressBar></ProgressBar>, div );
```

Comment:
Shouldn't you be using `render` here?
Also this test has no expect, I know you're not expecting anything to happen, which is why you should do an `expect.not.toThrow`, using `render` from React testing library.

Reply:
I added this test based on https://create-react-app.dev/docs/running-tests/#testing-components. That said, that section also mentioned the following:

> If you haven’t decided on a testing strategy yet, we recommend that you start with creating basic smoke tests for your components:

```
import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});
```

Given that I have other tests that are calling the `<ProgressBar />` component, I went ahead and removed the `renders without crashing` test cases from both test suits.

Reply:
It's true that this test is redundant, my question was mostly around using `document.createElement` and `ReactDOM.render` instead of react-testing-library render function.